ql/src/test/results/clientpositive/llap/constprog_dpp.q.out

```diff
@@ -84,12 +84,13 @@ Stage-0
                         Select Operator [SEL_40] (rows=1 width=4)
                           Output:["_col0"]
                           TableScan [TS_24] (rows=1 width=4)
-                            Output:["id"]
+                            default@tb2,tb2,Tbl:COMPLETE,Col:NONE,Output:["id"]
```

Comment:
I guess this was actually unrelated to current patch? Probably due to some data structure not being serialized when different union branches are copied?

Reply:
Yeah I think this is likely side effect of some changes in w.r.t serialization/de-serialization. Although this is positive side effect now that we have more information in explain plan.